.golangci.yml

```diff
@@ -39,28 +55,73 @@ linters:
   # even longer by marking them as 'nolint'.
   - lll
 
-  # We don't care (enough) about misaligned structs to lint that.
-  - maligned
+  # We want to allow short variable names.
+  - varnamelen
+
+  # We want to allow TODOs.
+  - godox
```

Comment:
Is this specific to VS code?

Reply:
Not sure what you mean. This is the description of the `godox` linter:
> Tool for detection of FIXME, TODO and other comment keywords

And since our code is full with TODOs, we don't want these warnings.